config/analysis-pipeline.json

```diff
@@ -289,15 +326,15 @@
   {
     "type": "team",
     "name": "sum",
-    "outputPath": "counts.totalAlgaeDrops",
+    "outputPath": "counts.dropAlgae",
```

Comment:
For a team, counts.dropAlgae isn't used anywhere; so, this block could be removed. Doesn't hurt to have it.